liota/dccs/iotcc.py

```diff
@@ -339,6 +362,38 @@ def store_reg_entity_details(self, entity_type, entity_name, reg_entity_id):
                 json.dump(msg, f, sort_keys=True, indent=4, ensure_ascii=False)
             f.close()
 
+    def remove_reg_entity_details(self, entity_name, reg_entity_id):
+        if self._iotcc_json == '':
+            log.warn('iotcc.json file missing')
+            return
+        try:
+            with open(self._iotcc_json, 'r') as f:
+                msg = json.load(f)
+            f.close()
+        except IOError, err:
+            log.error('Could not open {0} file '.format(self._iotcc_json) + str(err))
+        log.debug('Remove {0}:{1} from iotcc.json'.format(entity_name, reg_entity_id))
+        if msg["iotcc"]["EdgeSystem"]["SystemName"]==entity_name and msg["iotcc"]["EdgeSystem"]["uuid"]==reg_entity_id:
+            del msg["iotcc"]["EdgeSystem"]
```

Comment:
If the edge system is removed from the json file, what will happen to devices will they still exist in json file or will they get removed as device package is unloaded.